pages/api/queue/[id]/position.ts

```diff
@@ -0,0 +1,47 @@
+import { MongoClient } from "mongodb";
+import { NextApiRequest, NextApiResponse } from "next";
+import { ApiError, Room } from "../../types";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method == "POST") {
+    const client = new MongoClient(process.env.MONGODB_URI!);
+    const roomId = req.query.id;
+    const activeVideoIndex = parseInt(req.query.activeVideoIndex as string);
```

Comment:
This is going to set activeVideoIndex to NaN if req.query.activeVideoIndex is undefined